api/source/types/UserType.ts

```diff
@@ -19,6 +19,10 @@ export const UserType = new GraphQLObjectType({
       type: GraphQLNonNull(GraphQLString),
       description: "User's name",
     },
+    role_id: {
+      type: GraphQLNonNull(GraphQLID),
+      description: "User's id rol",
+    },
     first_lastname: {
       type: GraphQLNonNull(GraphQLString),
```

Comment:
Do not need to get Role ID. Role filed has already been implemented above. Remove this unnecessary field.